netstandard/tools/targets/NETStandard.Library.Trimming.targets

```diff
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="TrimFiles" AssemblyFile="$(NETStandardToolsTaskDirectory)NETStandard.Tools.dll" />
+
+  <Target Name="_determineTrimPackageInputs">
```

Comment:
Targets are named with an underscore when you don't want people to try and sequence around them.  Microsoft.Common.(CurrentVersion.)targets uses this convention.